python/src/deltachat/account.py

```diff
@@ -695,28 +695,24 @@ def shutdown(self) -> None:
         if self._dc_context is None:
             return
 
-        self.stop_io()
-
-        self.log("remove dc_context references")
-
-        # if _dc_context is unref'ed the event thread should quickly
-        # receive the termination signal. However, some python code might
-        # still hold a reference and so we use a secondary signal
-        # to make sure the even thread terminates if it receives any new
-        # event, indepedently from waiting for the core to send NULL to
-        # get_next_event().
+        # mark the event thread for shutdown (latest on next incoming event)
         self._event_thread.mark_shutdown()
-        self._dc_context = None
+
+        # stop_io also causes an info event that will terminate the event thread
```

Comment:
Maybe say "that will interrupt dc_get_next_event()" or "that will interrupt event thread and cause its termination" instead. Info event itself does not terminate the loop.

Reply:
did so.